ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -355,7 +355,7 @@ objects:
           oc --request-timeout=5s get nodes && exit 1
 
           echo "Restore single etcd - /usr/local/bin/etcd-snapshot-restore.sh"
-          bastion_ssh core@${SURVIVING_MASTER_NODE} 'sudo -i /bin/bash -x /usr/local/bin/etcd-snapshot-restore.sh /root/assets/backup/etcd/member/snap/db $(cat /tmp/etcd_connstring)'
+          bastion_ssh core@${SURVIVING_MASTER_NODE} 'sudo -i /bin/bash -x /usr/local/bin/etcd-snapshot-restore.sh /var/lib/etcd/member/snap/db $(cat /tmp/etcd_connstring)'
 
```

Comment:
I think `./assets/backup/etcd` is actually right. https://github.com/openshift/machine-config-operator/pull/804#issuecomment-499296463

Reply:
/close